### ISSUE TITLE
Nouveau CA

### ISSUE THREAD

**Issue by DewiBrunet** [openfab-lab/openfab#244]
- [ ] ref to file .md or pullRequest #: .................................................  
*(à remplir quand l'issue est résolue)*
## contexte
Nouveau CA avec : Xavier, Odile et Dewi
Faut clarifier la chose et se mettre en règle pour avancer sur la suite!

## objectif
- [ ] Lettre de démissions de Pauline et Mathis
- [ ] Invitation à une AG
- [ ] Feuille d'émargement et PV de l'AG
- [ ] un document à part pour la nomination des gestionnaires.
- [ ] rafraichir les status?
https://github.com/openfab-lab/openfab/blob/master/_space/asbl/statuts/statuts.04.2016-OCR.md
- [ ] Déménagement du siège social à BX (Dépôt à l'ancien greffier à Dinant)
- [ ] Scan ID recto/verso + adresses de résidences des nouveaux gestionnaires

**Comment by DewiBrunet** [comment]
- [ ] Vision OF (why)
- [ ] Etablir le CO/teams/team leader

**Comment by satblip** [comment]
- [ ] Bien lire ce [document](https://justice.belgium.be/fr/themes_et_dossiers/associations_et_fondations/asbl/statuts) pour voir tout ce qui est nécessaire
- [ ] Remplir les formulaire 1 et 2 ci joint [Formulaire 1](http://www.ejustice.just.fgov.be/tsv_pub/formulaire1asbl.doc) [Formulaire 2](http://www.ejustice.just.fgov.be/tsv_pub/formulaire2asbl.doc)
- [ ] Les modifications doivent se déposer au greffe, ce n'est possible en ligne que pour les créations. Il faudra donc déposer au greffe physique (ou envoyer par poste, mais pour le moment ça craint la poste ^^ On peut envoyer par coursier => 12€). Vu que l'asbl est originaire de Dinant, ce sera au greffe de Dinant. (Après ce pourra être Bruxelles, car le siège social sera déménagé). Tout peux être fait en un seul dépot. MAIS ATTENTION, dans l'ordre du texte, le CA actuel doit d'abord changer le siège social, ensuite démission/nomination (oui, c'est de l'éclate de juristes...).

**Comment by DewiBrunet** [comment]
Ok merci!

Pour les status, je propose que je reprenne https://github.com/openfab-lab/openfab/blob/master/_space/asbl/statuts/statuts.04.2016-OCR.md avec modifications minimes (changer les noms, retaper un peu de notre nouvelle vision...), pour moi plus le temps d'essayer de faire un ROI propre ou mettre du lol.

Formulaire 1 et 2 j'y pige rien... c'est obligatoire?^^

Changer de siège c'est donc une priorité si je comprends bien? Si c'est "juste" payer 12€ pour envoyer une lettre qu'est ce qu'on attend? Comment faire?

@nicolasdb @satblip je veux avancer au plus vite la dessus sachant que c'est chiant mais très important. Je suis prêt à donner du temps mais je suis vraiment perdu dans toutes ces histoires... (et faut pas les prendre à la légère juridiquement il me semble).

DONC faut se diviser les tâches (avec Odile et Xavier aussi?) ou bien me dire clairement quoi faire sinon on va encore être capable de traîner ça en 2019.

**Comment by satblip** [comment]
Oui, c'est obligatoire, tu n'as pas le choix ^^ Je peux les remplir avec vous, je l'ai déjà pas mal fait ;)

**Comment by satblip** [comment]
1) RDV pour finaliser les documents
2) Signer ces documents avec tous les membres du CA
3) Les envoyer au greffe du tribunal (tu peux tout faire en un)

Tu dois commencer par l'étape 1 ;)

**Comment by DewiBrunet** [comment]
merci :) 
étape 1 en cours, Odile m'a répondu qu'elle est dispo en soirée semaine prochaine, je vois avec Xavier et je te dis quoi!

**Comment by DewiBrunet** [comment]
On a bien avancé avec @satblip :) 
Les formulaires 1 & 2 sont quasi remplis, manque les dates et info perso du nouveau CA (les docs sont sur le drive gestion).
Odile, Xavier et moi on se réunit ce mercredi 28/11 pour finaliser ça.
@nicolasdb  niveau lettre de démissions de Pauline et Mathis on en est ou?

**Comment by nicolasdb** [comment]
Il faut qu'on commence à organiser les task dans des issues séparées et à un endroit donné. 

Est-ce le bon endroit ici, sur ce repo? _Le repo Openfab, je le verrai plus orienté gestion du lab, la communauté._ 
Est-ce qu'on travaillerai directement sur le repo RTFM? pour une publication des résultats dans le RTFM, _j'aimerai qu'on assume cette transparence_ 
Est-ce qu'on crée un repo (privé/publique?) pour le CA, les stratégies et les tâches à accomplir?

Qu'en pensez vous?
